src/common/utility/cmdlib.h

```diff
@@ -35,7 +35,7 @@ char(&_ArraySizeHelper(T(&array)[N]))[N];
 bool FileExists (const char *filename);
 bool DirExists(const char *filename);
 bool DirEntryExists (const char *pathname, bool *isdir = nullptr);
-bool GetFileInfo(const char* pathname, size_t* size, time_t* time);
+bool GetFileInfo(const char* pathname, size_t* size);
```

Comment:
Raze [uses](https://github.com/coelckers/Raze/blob/a00da236541fddc6f9c96ebfe410e681e3180918/source/core/searchpaths.cpp#L304) the third argument of `GetFileInfo()` function.